feature/common/src/main/java/com/yapp/itemfinder/feature/common/datalist/adapter/AddItemImagesViewHolder.kt

```diff
@@ -0,0 +1,70 @@
+package com.yapp.itemfinder.feature.common.datalist.adapter
+
+import com.yapp.itemfinder.domain.model.AddItemImages
+import com.yapp.itemfinder.feature.common.R
+import com.yapp.itemfinder.feature.common.adaper.AddItemImagesInnerAdapter
+import com.yapp.itemfinder.feature.common.adaper.AddItemImagesInnerData
+import com.yapp.itemfinder.feature.common.databinding.AddItemImagesBinding
+import com.yapp.itemfinder.feature.common.utility.AddImagesItemDecoration
+
+class AddItemImagesViewHolder(
+    val binding: AddItemImagesBinding
+) : DataViewHolder<AddItemImages>(binding) {
+    lateinit var mAdapter: AddItemImagesInnerAdapter
+
+
+    private var prevData: AddItemImages? = null
+    private var dataChanged = false
+    private val context = binding.root.context
+    private val addImagesDecoration by lazy {
+        AddImagesItemDecoration(context.resources.getDimension(R.dimen.add_images_recyclerview_margin).toInt())
+    }
+    init {
+        binding.innerRecyclerView.addItemDecoration(addImagesDecoration)
+    }
+
+    override fun reset() {
```

Comment:
아래와 같이 처리해도 될 것 같습니다~
```suggestion
    override fun reset() = Unit
```